src/PrivateDns/PrivateDns/help/Set-AzPrivateDnsZone.md

```diff
@@ -0,0 +1,228 @@
+---
+external help file: Microsoft.Azure.PowerShell.Cmdlets.PrivateDns.dll-Help.xml
+Module Name: Az.PrivateDns
+online version: https://docs.microsoft.com/en-us/powershell/module/az.privatedns/set-azprivatednszone
+schema: 2.0.0
+---
+
+# Set-AzPrivateDnsZone
+
+## SYNOPSIS
+Updates a Private DNS zone from a resource group.
+
+## SYNTAX
+
+### Fields (Default)
+```
+Set-AzPrivateDnsZone -ResourceGroupName <String> -Name <String> [-Tags <Hashtable>]
+ [-DefaultProfile <IAzureContextContainer>] [-WhatIf] [-Confirm] [<CommonParameters>]
+```
+
+### ResourceId
+```
+Set-AzPrivateDnsZone [-Tags <Hashtable>] -ResourceId <String> [-DefaultProfile <IAzureContextContainer>]
```

Comment:
For every cmdlets, the order of parameter should be: identity from top level down, all other mandatory parameters, then all non-mandatory parameters. The way to fix the order here is to order the parameters in cs files in the correct order.  Please fix everywhere and regen the help files.

Reply:
fixed it.